api/controllers/app/install/controller.go

```diff
@@ -205,6 +212,7 @@ func NewInstallController(opts ...InstallControllerOption) (*InstallController,
 			appreleasemanager.WithReleaseData(controller.releaseData),
 			appreleasemanager.WithLicense(license),
 			appreleasemanager.WithPrivateCACertConfigMapName(controller.privateCACertConfigMapName),
+			appreleasemanager.WithK8sVersion(controller.k8sVersion),
```

Comment:
this is only passing the version to the release manager, not the install manager. is that correct?

Reply:
updated so they both use it